### PR TITLE
Remove raspilot

### PR DESCRIPTION
**Pull request by lucasdemarchi**
Follow up from https://discuss.ardupilot.org/t/remove-unused-boards/19711

I also contacted @pritamghanghas who was the last contact for raspilot I had and he agreed with it.

### CODE REVIEW

File: libraries/AP_HAL_Linux/HAL_Linux_Class.cpp [hunk 1]
```diff
@@ -194,13 +180,17 @@ static RCOutput_PCA9685 rcoutDriver(i2c_mgr_instance.get_device(1, PCA9685_PRIMA
 static RCOutput_PCA9685 rcoutDriver(i2c_mgr_instance.get_device(1, PCA9685_QUATENARY_ADDRESS), false, 0, RPI_GPIO_4);
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_DARK
 static RCOutput_PCA9685 rcoutDriver(i2c_mgr_instance.get_device(1, PCA9685_QUINARY_ADDRESS), false, 0, RPI_GPIO_27);
+<<<<<<< HEAD
```
Comment (OXINARF) [inline libraries/AP_HAL_Linux/HAL_Linux_Class.cpp]: Ups

File: libraries/AP_HAL_Linux/HAL_Linux_Class.cpp [hunk 2]
```diff
@@ -194,13 +180,17 @@ static RCOutput_PCA9685 rcoutDriver(i2c_mgr_instance.get_device(1, PCA9685_PRIMA
...
 /*
  use the STM32 based RCOutput driver on Raspilot
  */
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_RASPILOT
 static RCOutput_Raspilot rcoutDriver;
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ZYNQ || \
       CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_OCPOC_ZYNQ
+=======
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ZYNQ
+>>>>>>> AP_HAL_Linux: remove raspilot
```
Comment (OXINARF) [inline libraries/AP_HAL_Linux/HAL_Linux_Class.cpp]: Ups

**Comment by OXINARF** [comment]
You also need to remove it from Travis.

**Comment by OXINARF** [comment]
@lucasdemarchi You managed to update it so many times and still not remove it from Travis 😄

**Comment by lucasdemarchi** [comment]
are you sure? check again :)

**Comment by OXINARF** [comment]
Ahahah you are lucky you are fast 😛

**Pull request merged**